packages/itwinui-react/src/utils/components/VirtualScroll.tsx

```diff
@@ -118,21 +131,65 @@ export type VirtualScrollProps = {
  *    This is my item #{index}
  *  </div>
  * ), [])
+ * <div style={{overflow: 'auto'}} ref={ref}>
  * <VirtualScroll
```

Comment:
The `VirtualScroll` component is being removed in the [`Tree`](https://github.com/iTwin/iTwinUI/pull/2091) component's PR, so I don't think it makes much sense to update the docs for this component. I realized that the PR description was out of date, and referenced some things that were previously being done with this component that have since been changed. I have updated the PR description to mention that `VirtualScroll` is being removed.